obs-vscode-extension/package.json

```diff
@@ -17,12 +17,14 @@
         "@types/mocha": "^2.2.42",
         "@types/node": "^10.14.17",
         "tslint": "^5.12.1",
-        "typescript": "^3.3.1",
-        "vscode": "^1.1.34"
+        "typescript": "^3.3.1"
```

Comment:
I had typescript 3.5.2 and your code refused to compile because you used the ?. operator, which is new in 3.7. You should update the dependencies accordingly (I've upgraded typescript locally, but package.json should be right).